app/javascript/react/containers/PlacesShowContainer.js

```diff
@@ -23,17 +25,36 @@ class PlacesShowContainer extends Component {
     })
     .then(response => response.json())
     .then(body => {
-      this.setState( {place: body.place} )
+      this.setState( {place: body.place, reviews: body.place.reviews} )
```

Comment:
If you do things this way, then you have all the reviews in state twice: once nested under place, and once as their own thing. I'd recommend removing this change and accessing the reviews as `this.state.place.reviews`.

Reply:
We were getting a 'cannot .map on undefined' error when we did that because on the first render this.state.place.reviews doesn't exist. The above was our work around. Do you have a suggestion on how to better structure as to avoid that error?